### ISSUE TITLE
Error when training the pano_outpainting model

### ISSUE THREAD

**Issue by Primer-1-wy** [Tangshitao/MVDiffusion#53]
Hi,I'm trying to train the pano_outpainting model. I successfully generate image when I set resolution to 256*256.However I failed to reproduce the result when the resolution is 512*512.The error img like below:
![4909_0_2_pred](https://github.com/user-attachments/assets/7ef5a953-5177-44e7-9ef3-5524f6b202dd)

**Comment by Tangshitao** [comment]
Hi, have you tried our inference codes for outpainting?

**Comment by Primer-1-wy** [comment]
> Hi, have you tried our inference codes for outpainting?

Yes,I tried. It's okay to use the same trained model for inference. But I wanted to retrain, but something went wrong, I think the model wasn't trained well.
